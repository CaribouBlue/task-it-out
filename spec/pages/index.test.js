import React from 'react';
import renderer from 'react-test-renderer';
import IndexPage from '../../pages/index';

test('Index page matches snapshot', () => {
  const component = renderer.create(
    <IndexPage></IndexPage>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
