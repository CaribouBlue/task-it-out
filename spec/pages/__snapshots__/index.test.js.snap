// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page matches snapshot 1`] = `
<div
  className="grid-y medium-grid-frame"
>
  <div
    className="cell shrink top-bar"
  >
    <div
      className="top-bar-left"
    >
      <ul
        className="menu"
      >
        <li
          className="menu-text"
        >
          Task It Out
        </li>
      </ul>
    </div>
    <div
      className="top-bar-right"
    >
      <ul
        className="menu"
      />
    </div>
  </div>
  <div
    className="cell auto cell-block-container"
  >
    <h1>
      Page title
    </h1>
    <p>
      page content
    </p>
  </div>
  <footer
    className="cell shrink"
  >
    <p>
      footer
    </p>
  </footer>
</div>
`;
